src/components/VmModals/DeleteConfirmationModal.js

```diff
@@ -7,18 +7,24 @@ class DeleteConfirmationModal extends React.Component {
   constructor (props) {
     super(props)
     this.state = { show: false }
+    this.handleTriggerClick = this.handleTriggerClick.bind(this)
     this.handleClose = this.handleClose.bind(this)
     this.handleDelete = this.handleDelete.bind(this)
   }
 
+  handleTriggerClick (e) {
+    if (e && e.preventDefault) e.preventDefault()
```

Comment:
Same question as above, dose this check really need?

Reply:
When the component does not know for certain what the `trigger` component will be, and it will install an `onClick` handler, it makes sense to me to cover the edge cases and prevent an `<a href='#'>...</a>` type of tag from changing the browser URL.

Reply:
Now that we're on the same page, I'll take out the check here as well.